std/internal/entropy/common.d

```diff
@@ -0,0 +1,181 @@
+// Written in the D programming language.
+
+/+
+    Non-backend-specific functionality.
+
+    License:   $(HTTP www.boost.org/LICENSE_1_0.txt, Boost License 1.0).
+    Authors:   Elias Batek
+    Source:    $(PHOBOSSRC std/internal/entropy/common.d)
+ +/
+module std.internal.entropy.common;
+
+@nogc nothrow:
+
+///
+enum EntropySource
```

Comment:
no documented example(?), I'm not sure what the style check wants here. There is nothing to unittest. It's a `enum`.

Reply:
I’ve decided to disable this style check: <https://github.com/dlang/phobos/pull/10748/commits/cfcb0ced23a6f3168e836c647a72e6f2069e75f9>
Reasoning: <https://github.com/dlang/phobos/pull/10748#pullrequestreview-2810627921>